README.md

```diff
@@ -40,9 +47,52 @@ Here are some key features of YuniKorn.
 - Customized resource types (like GPU) scheduling support.
 - Rich placement constraints support.
 - Automatically map incoming container requests to queues by policies. 
-- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management. 
+- Node partition: partition cluster to sub-clusters with dedicated quota/ACL management.
+
+### Integration with K8s:
+
+`yunikorn-k8shim` nicely integrated with K8s, supported features like: 
+
+- _Predicates:_ All kinds of predicates such as node-selector, pod affinity/anti-affinity, taint/tolerant, etc.
+- _Persistent volumes:_ We have verified hostpath, EBS, NFS, etc. 
+- _K8s namespace awareness:_ YuniKorn support hierarchical of queues, does it mean you need to give up K8s namespace? Answer is no, by default YuniKorn fully support automatically
+ map K8s
+ namespaces to YuniKorn queues. All Namespace related ResourceQuota, permissions will be still valid.
+- _Metrics:_ Prometheus, Grafana integration.
+- _Cluster AutoScaler_: YuniKorn can nicely work with Cluster AutoScaler (https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) to drive cluster scales up and down.
+- _K8s Events_: YuniKorn also integrated with K8s events, so lots of information can be retrieved by using `kubectl describe pod`.
+
+#### Performance testing
+We love high-performance software, and we made tremendous efforts to make it to support large scale cluster and high-churning tasks. 
+Here's [Performance Test Result](docs/evaluate-perf-function-with-Kubemark.md) 
+
+#### Deployment model
+Yunikorn can be deployed as a K8s custom scheduler, and take over all POD scheduling. Or, YuniKorn can be deploy together with K8s default scheduler and take over
+ subset of nodes.
+ 
+#### Verified K8s versions 
+
+| K8s Version   | Support?  |
+| ------------- |:-------------:|
+| 1.12.x (or earlier) | X |
+| 1.13.x | √ |
```

Comment:
Are we sure I thought we had moved to 1.14 a long time ago and cannot remember the last time we have used the 1.13 libs for a build

Reply:
We're still running YuniKorn + EKS 1.13, since EKS 1.13 is still not deprecated and have many users on that, we can claim it is supported.